spec/Memory_Consistency.tex

```diff
@@ -761,6 +778,8 @@ \section{Discussion}
 
 \begin{chapelexample}{atomicSpinWait.chpl}
 Atomic variables provide an alternative means to spin-wait. Atomic variables include a waitFor method that will block the calling thread until a read of the atomic value matches a particular value. For example:
+\todo{Why introduce waitFor(), instead of using the direct translation to
+atomic vars?}
```

Comment:
because waitFor includes the appropriate yelds/network servicing. Generally speaking, it's a bad idea to put a tight spin-loop in a cooperatively scheduled environment.